copyright_and_licenses.csv

```diff
@@ -166,7 +166,6 @@ nc_dittt_gr3dnc02b.tif,Gouvernement de Nouvelle Calédonie - DITTT,Open License
 nc_dittt_gr3dnc03a.tif,Gouvernement de Nouvelle Calédonie - DITTT,Open License France - https://www.etalab.gouv.fr/wp-content/uploads/2014/05/Open_Licence.pdf,,
 nc_dittt_Ranc08_Circe.tif,Gouvernement de Nouvelle Calédonie - DITTT,Open License France - https://www.etalab.gouv.fr/wp-content/uploads/2014/05/Open_Licence.pdf,,
 nc_dittt_README.txt,Disclaimed,Public domain,,
-nl_nsgi_naptrans2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),CC-BY-4.0,,
```

Comment:
that line should be kept, but the "version_removed" last column set to 1.10

Reply:
Thanks for fixing my misunderstanding.